i18n/en.json

```diff
@@ -2666,6 +2671,9 @@
   "posts_view.maxLoaded": "Looking for a specific message? Try searching for it",
   "posts_view.newMsg": "New Messages",
   "posts_view.newMsgBelow": "New {count, plural, one {message} other {messages}}",
+  "promote_to_user_modal.desc": "This action promotes the guest {username} to a member. This action will allow the user to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to promote guest {username} to user?",
```

Comment:
nit: same

```suggestion
  "promote_to_user_modal.desc": "This action promotes the guest {username} to a member. It will allow the user to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to promote guest {username} to user?",
```